README.md

```diff
@@ -1,2 +1,29 @@
-# Samples
-Example pointcloud data that can be used for example displays and tests.
+# Testing Data (for point cloud visualization)
+
+Example point cloud data that can be used for example displays and tests.
+
+**NOTE:** Creating these datasets requires [PDAL](https://pdal.io). If you don't have pdal installed, please checkout their [installation guides]((https://pdal.io/download.html)) or use via [docker](https://pdal.io/quickstart.html).
+
+## Data description
+As of now, there are two sets of point cloud data that are available for testing. Basic details are as follows:
+
+| Data Set      | Creator                             | License        | Formats           |
```

Comment:
Having to maintain this list is gonna be bothersome and an opening for inconsistencies, maybe keep that information inside the folders only?

Reply:
sounds good